db/migrate/20151015190030_create_sponsor_group.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSponsorGroup < ActiveRecord::Migration
+  def change
+    create_table :sponsor_groups do |t|
+      t.string :name
+      t.text :address
```

Comment:
We'll probably need to break this out into fields
ie: address1, address2, city, postal code, ..